test/test_bulk_renewal.js

```diff
@@ -62,13 +63,15 @@ contract('ETHRegistrarController', function (accounts) {
 		baseRegistrar = await BaseRegistrar.new(ens.address, namehash.hash('eth'), {from: ownerAccount});
 
 		// Set up a dummy price oracle and a controller
-		priceOracle = await SimplePriceOracle.new(1);
+        const dummyOracle = await DummyOracle.new(toBN(1000000000000000000));
+        priceOracle = await StablePriceOracle.new(dummyOracle.address, [1]);
 		controller = await ETHRegistrarController.new(
 			baseRegistrar.address,
 			priceOracle.address,
 			600,
 			86400,
 			{from: ownerAccount});
+		await controller.setPriceOracle(priceOracle.address, {from: ownerAccount})
```

Comment:
This does not seem to add anything to this test as priceOracle is already passed as initial argument of constructor (I commented out this line and the test still passed). Please either remove this line or add additional assertion to test this logic